web/src/pages/report/SqlQueryUI.tsx

```diff
@@ -0,0 +1,440 @@
+
+            {dbSetupStatus.tableSetupStatus.map((table) => (
+                <Fragment key={table.name}>
+                    <ListItem disablePadding>
+                        <ListItemButton
+                            onClick={() => handleSidebarSectionToggle(table.name)}
+                            selected={selectedTableName === table.name}
+                        >
+                            <ListItemText>
+                                <Typography component={'span'}>{table.name}</Typography>
+                            </ListItemText>
+                            {table.status === 'error' && <ErrorIcon color="error" />}
+                            {table.status === 'success' && (
+                                <Link
+                                    to={`/project/${projectName}/query/${table.name}`}
+                                    onClick={(e) => {
+                                        e.stopPropagation()
+                                    }}
+                                >
+                                    <IconButton aria-label="view table" size={'small'}>
+                                        <EyeCon />
+                                    </IconButton>
+                                </Link>
+                            )}
+                            {expandedTables.includes(table.name) ? <ExpandLess /> : <ExpandMore />}
+                        </ListItemButton>
+                    </ListItem>
+
+                    <Collapse in={expandedTables.includes(table.name)} timeout={150} unmountOnExit>
+                        {table.status === 'success' && (
+                            <List component="div" disablePadding dense={true}>
+                                {table.tableInfo.columns.map((column) => (
+                                    <ListItemText
+                                        key={column.columnName}
+                                        sx={{ pl: 4, py: 0.2, whiteSpace: 'nowrap' }}
+                                    >
+                                        <Typography component={'span'} noWrap>
+                                            {column.columnName}
+                                        </Typography>
+                                        <Chip
+                                            label={column.columnType}
+                                            size="small"
+                                            sx={{ fontSize: 10, p: 0, ml: 1 }}
+                                        />
+                                    </ListItemText>
+                                ))}
+                            </List>
+                        )}
+                        {table.status === 'error' && (
+                            <Alert severity="error">{table.errorMessage}</Alert>
+                        )}
+                    </Collapse>
+                </Fragment>
+            ))}
+        </List>
+    )
+
+    const sidebarWidth = 250
```

Comment:
I would probably make numeric constant as all upper case and put it on the top of the file, but not a biggie.

Reply:
Ah good pickup, that was some prototype code that slipped through :)
fixed now